docker/postgres/initdb.sql

```diff
@@ -0,0 +1,14 @@
+-- Create the telemetry user
+CREATE USER telemetry;
+
+-- Create staging database and grant telemetry user access
+CREATE DATABASE staging WITH TEMPLATE template0;
+GRANT ALL PRIVILEGES ON DATABASE staging TO telemetry;
+
+-- Create operational database and grant telemetry user access
+CREATE DATABASE operational WITH TEMPLATE template0;
+GRANT ALL PRIVILEGES ON DATABASE operational TO telemetry;
+
+-- Create telemetry database and grant telemetry user access
+CREATE DATABASE telemetry WITH TEMPLATE template0;
+GRANT ALL PRIVILEGES ON DATABASE telemetry TO telemetry;
```

Comment:
Is it worth to have a group for this instead of giving the permissions inside the db directly? Are we ever going to have more than one user on this db?

Reply:
For now I'm keeping it simple.

In the long run these should probably be configured to be equivalent to the setup that an Amazon RDS uses.

Reply:
Created #33 to track the intent to do this.